templates/scenarios/common/init-debug-vsc-bot/{%dotVscodeFolderName%}/tasks.json

```diff
@@ -85,7 +85,7 @@
         {
             "label": "Start bot",
             "type": "shell",
-            "command": "node teamsfx/script/run.js . teamsfx/.env.local",
+            "command": "node script/run.js . teamsfx/.env.local",
```

Comment:
Still `teamsfx/.env.local`?

Reply:
The `.env` refactor is still under discussion... so this PR only moves app yaml and script